packages/docs/pages/form.tsx

```diff
@@ -27,195 +34,310 @@ import {
-
-                const handleSubmit = (event) => {
-                  const form = event.currentTarget;
-
-                  if (form.checkValidity() === false) {
-                    event.preventDefault();
-                    event.stopPropagation();
-                  }
-                };
-
-                const handleChange = (event) => {
-                  const { target } = event;
-                  const regex = RegExp(target.pattern, 'g');
-
-                  regex.test(target.value) ? setError('') : setError(ERROR_MSG);
-
-                  setValue(target.value);
-                };
-
-                return (
-                  <Form onSubmit={handleSubmit}>
-                    <FormGroup>
-                      <Input
-                        label="Example"
-                        description="Remove characters to preview validation."
-                        value={value}
-                        error={error}
-                        onChange={handleChange}
-                        pattern="^.{1,3}$"
-                        required
-                      />
-                    </FormGroup>
-                    <FormGroup>
-                      <Input label="City" error="You must enter a valid City." placeholder="Austin" required />
-                      <Input label="State" placeholder="Texas" required />
-                      <Input
-                        label="Postal Code"
-                        error="You must enter a valid Postal Code."
-                        placeholder="78726"
-                        required
-                      />
-                    </FormGroup>
-                  </Form>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => (
-        <>
-          <FormPropTable inheritedProps={<MarginPropTable collapsible />} />
-          <FormErrorPropTable id="error" />
-          <FormLabelPropTable id="label" />
-          <FormDescriptionPropTable id="label" />
-          <FormGroupPropTable />
-          <FormFieldsetPropTable />
-        </>
-      ),
-    },
-  ];
-
   return (
     <>
       <H1>Form</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          The provided <Code primary>Form</Code> component is a wrapper around an{' '}
+          <Link
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form"
+            external
+            target="_blank"
+            rel="nofollow noreferrer"
+          >
+            HTML <Code>{'<form />'}</Code>
+          </Link>{' '}
+          element. It provides form width styling and accessibility features when using form controls within.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>Any usage of a form control</List.Item>
```

Comment:
Consistent with other pages.

```suggestion
          <List.Item>Any usage of a form control.</List.Item>
```